Description: <Add sunway support>
Author: Miao Changwei <miaochangwei@uniontech.com>

---
Last-Update: 2022-04-20

--- alsa-lib-1.2.5.1.orig/include/alsa-symbols.h
+++ alsa-lib-1.2.5.1/include/alsa-symbols.h
@@ -47,7 +47,7 @@
 		default_symbol_version(real, name, version)
 #else
 #define use_symbol_version(real, name, version) /* nothing */
-#if defined(__alpha__) || defined(__mips__)
+#if defined(__alpha__) || defined(__mips__) || defined(__sw_64__)
 #define use_default_symbol_version(real, name, version) \
         __asm__ (".weak " ASM_NAME(#name)); \
         __asm__ (ASM_NAME(#name) " = " ASM_NAME(#real))
